blueprints/data-solutions/vertex-mlops/variables.tf

```diff
@@ -0,0 +1,228 @@
+    internal_ip_only      = bool
+    idle_shutdown_timeout = bool
+  }))
+  default = null
+}
+
+variable "org_policies" {
+  description = "Org-policy overrides at project level."
+  type = map(object({
+    inherit_from_parent = optional(bool) # for list policies only.
+    reset               = optional(bool)
+
+    # default (unconditional) values
+    allow = optional(object({
+      all    = optional(bool)
+      values = optional(list(string))
+    }))
+    deny = optional(object({
+      all    = optional(bool)
+      values = optional(list(string))
+    }))
+    enforce = optional(bool, true) # for boolean policies only.
+
+    # conditional values
+    rules = optional(list(object({
+      allow = optional(object({
+        all    = optional(bool)
+        values = optional(list(string))
+      }))
+      deny = optional(object({
+        all    = optional(bool)
+        values = optional(list(string))
+      }))
+      enforce = optional(bool, true) # for boolean policies only.
+      condition = object({
+        description = optional(string)
+        expression  = optional(string)
+        location    = optional(string)
+        title       = optional(string)
+      })
+    })), [])
+  }))
+  default  = {}
+  nullable = false
+}
+
+variable "organization_domain" {
+  description = "Organization domain."
+  type        = string
+}
+
+variable "prefix" {
+  description = "Prefix used for the project id."
+  type        = string
+  default     = null
+}
+
+variable "project_create" {
+  description = "Create project. When set to false, uses a data source to reference existing project."
+  type        = bool
+  default     = true
+}
+
+variable "project_id" {
+  description = "Project id."
+  type        = string
+}
+variable "project_services" {
+  description = "List of core services enabled on all projects."
+  type        = list(string)
+  default = [
+    "aiplatform.googleapis.com",
+    "artifactregistry.googleapis.com",
+    "bigquery.googleapis.com",
+    "cloudbuild.googleapis.com",
+    "compute.googleapis.com",
+    "datacatalog.googleapis.com",
+    "dataflow.googleapis.com",
+    "iam.googleapis.com",
+    "monitoring.googleapis.com",
+    "notebooks.googleapis.com",
+    "secretmanager.googleapis.com",
+    "servicenetworking.googleapis.com",
+    "serviceusage.googleapis.com"
+  ]
+}
+
+variable "region" {
+  description = "Region used for regional resources."
+  type        = string
+  default     = "europe-west4"
+}
+
+variable "repo_name" {
+  description = "Cloud Source Repository name. null to avoid to create it."
+  type        = string
+  default     = null
+}
+
+variable "vpc" {
+  description = "Shared VPC configuration for the project."
+  type = object({
+    host_project = string
+    gke_setup = object({
+      enable_security_admin     = bool
+      enable_host_service_agent = bool
+    })
+    subnets_iam = map(list(string))
+  })
+  default = null
+}
+
+variable "vpc_local" {
```

Comment:
I would reduce the variable surface: setting the name of the VPC is not relevant for the blueprint, I would hardcode it as in other blueprints

Reply:
done